### PR TITLE
Safari pull-scrolling prevents touch scroll on mobile

### PR DESCRIPTION
**Pull request by TobiaszCudnik**
This PR fixes pull scrolling issues on iOS by disabling bouncing on body and assuming the bottom status bar is always visible.

Other browser's shouldn't be affected.

### CODE REVIEW

File: src/containers/wallet/Wallet.tsx [hunk 1]
```diff
@@ -112,6 +121,13 @@ class Wallet extends React.Component<DecoratedProps, State> {
   }
 
   componentWillMount() {
+    // store info if the current platform is supported
+    if (typeof inobounce.supported === 'undefined') {
+      inobounce.supported = inobounce.isEnabled();
+    }
+    if (inobounce.supported) {
+      inobounce.enable();
```
Comment (roosmaa) [inline src/containers/wallet/Wallet.tsx]: From iNoBounce README/docs:

> Enable iNoBounce. It's enabled by default on platforms that support -webkit-overflow-scrolling, so you only need to call this method if you explicitly disable it or want to enable it on a platform that doesn't support -webkit-overflow-scrolling.

If the above is to be believed the code blocks in componentWillMount and componentWillUnmount seem redundant OR there's a bug here that the iNoBounce is enabled for non-Wallet components as well by default.

**Comment by roosmaa** [comment]
Could you also make a quick screen grab (iirc QuickTime could record connected iPhone screens) regarding what this bounce effect is that this PR is trying to remove?

File: src/containers/wallet/Wallet.tsx (hunk 1 above)
Comment (TobiaszCudnik) [reply src/containers/wallet/Wallet.tsx]: The key phrase here is:
> or want to enable it on a platform that doesn't support -webkit-overflow-scrolling

Once disabled, its enabled for every platform, thats why the result of autodetection is stored before calling `enable`.

I'll move `inobounce.supported = inobounce.isEnabled();` to `App.tsx` so it'll looks more global.

**Comment by TobiaszCudnik** [comment]
Videos from [before](https://drive.google.com/open?id=1OlGgFICsq4dDBMyi63EMhDzA6IoarK8-) and [after](https://drive.google.com/open?id=1omW440ohdGl71xrMcweRru94_j_GLwgZ).

Safari scroll bouncing is a common problem for fullscreen apps, especially on iOS.

**Comment by roosmaa** [comment]
Yeah, I spent quite a lot of time figuring out the way to prevent it.. but apparently it only works on Android :( so yeah, even though there's a slight visual artifact on the after video for the onboarding scrolling, I guess we can fix those things later.

**Pull request merged**